src/owncloud/apache_config

```diff
@@ -14,3 +15,14 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+# MPM Event
+<IfModule event.c>
+	StartServers			2
+	MinSpareThreads			4
+	MaxSpareThreads			32
+	ThreadsPerChild			16
+	ServerLimit				4
+	MaxRequestWorkers		64
+	MaxConnectionsPerChild	0
+</IfModule>
```

Comment:
Actually, I had forgotten that this only works with PHP-FPM, so it's best to keep a sane starting point here, which can be tuned later, depending on when and if we agree on a better config.